src/site/content/en/blog/smoothness/index.md

```diff
@@ -0,0 +1,563 @@
+Animations](https://developers.google.com/web/fundamentals/design-and-ux/animations/css-vs-javascript),
+and ensuring you [animate composite
+properties](https://developer.chrome.com/blog/hardware-accelerated-animations/),
+is a great initial step to ensuring smooth and efficient animations. But still,
+this alone does not guarantee smoothness because even efficient web animations
+have performance limits. That's why it is always important to measure!
+
+## What are animation frames?
+
+Updates to the visual representation of a page take time to appear. Visual
+changes will eventually lead to a new animation frame, which is eventually
+presented to the user via their display.
+
+Generally, displays can only update at certain intervals, so visual updates are
+batched. Many displays update on a fixed interval of time, such as 60 times a
+second (that is 60&nbsp;Hz). Some more modern displays can offer higher refresh rates
+(90-120&nbsp;Hz are becoming common). Often these displays can actively adapt between
+refresh rates as needed, or even offer fully variable frame rates.
+
+The goal for any application, like a game or a browser, is to process all these
+batched visual updates and produce a visually complete animation frame within
+the deadline, every time. Note that this goal is entirely distinct from other
+important browser tasks such as loading content from the network quickly or
+executing JavaScript tasks efficiently.
+
+At some point, it can become too difficult to complete all visual updates within
+the allotted deadline assigned by the display. When this happens, the browser
+**drops a frame**. Your screen doesn't go black, it just repeats itself. You see
+the same visual update for a bit longer—the same animation frame that was
+presented at the previous frame opportunity.
+
+This actually happens often! It is not necessarily even perceptible, especially
+for static or document-like content, which is common on the web platform in
+particular. Dropped frames only become apparent when there are important visual
+updates, such as animations, for which we need a steady stream of animation
+updates to show smooth motion.
+
+## What impacts animation frames?
+
+Web developers can greatly impact the ability of a browser to quickly and
+efficiently render and present visual updates!
+
+Some examples:
+
+- Using content that is too large or resource intensive to decode quickly on the
+  target device.
+- [Using too many
+  layers](https://developers.google.com/web/fundamentals/performance/rendering/stick-to-compositor-only-properties-and-manage-layer-count)
+  requiring too much GPU memory.
+- Defining overly complex CSS styles or web animations.
+- Using design anti-patterns that disable fast rendering optimizations.
+- Too much JS work on the main thread, leading to long tasks that block certain
+  visual updates.
+
+But how can you know when an animation frame has slipped its deadline and caused
+a dropped frame?
```

Comment:
```suggestion
But how can you know when an animation frame has missed its deadline and caused
a dropped frame?
```

Reply:
Some of these edits may seem a bit finicky, but some are to improve readability for those who English isn't a first language. "Missed a deadline" is more in the common vernacular than "slipped a deadline".